src/t8_forest/t8_forest_general.h

```diff
@@ -774,14 +776,17 @@ t8_forest_element_point_inside (t8_forest_t forest, t8_locidx_t ltreeid, const t
  *                              lies within an \a element, false otherwise. The return value is also true if the point 
  *                              lies on the element boundary. Thus, this function may return true for different leaf 
  *                              elements, if they are neighbors and the point lies on the common boundary.
+ * \param [in]      geom_is_axis_aligned Flag to tell if the used geometry is an axis-aligned geometry. Simplifies the
+ *                             check for this type of geometry. Can only be used for lines, quads and hexs. 
  * \param [in]      tolerance   Tolerance that we allow the point to not exactly match the element.
  *                              If this value is larger we detect more points.
  *                              If it is zero we probably do not detect points even if they are inside
  *                              due to rounding errors.
  */
 void
 t8_forest_element_point_batch_inside (t8_forest_t forest, t8_locidx_t ltreeid, const t8_element_t *element,
```

Comment:
How about calling this function `t8_forest_element_points_inside` ?

Reply:
I would also consider to remove `t8_forest_element_point_inside` in the midterm future.

Reply:
That would be okay for me, to change the name to points, and remove the function for a single point.